Frends.Community.Email/EmailTask.cs

```diff
@@ -309,13 +309,20 @@ private static MimeMessage CreateMimeMessage([PropertyTab] Input message)
             mail.Subject = message.Subject;
 
             // Add recipients.
-            foreach (var recipientAddress in recipients) mail.To.Add(MailboxAddress.Parse(recipientAddress));
+            foreach (var recipientAddress in recipients)
+                mail.To.Add(MailboxAddress.Parse(recipientAddress));
 
             // Add CC recipients.
-            foreach (var ccRecipient in ccRecipients) mail.Cc.Add(MailboxAddress.Parse(ccRecipient));
+            foreach (var ccRecipient in ccRecipients)
+                mail.Cc.Add(MailboxAddress.Parse(ccRecipient));
 
             // Add BCC recipients.
-            foreach (var bccRecipient in bccRecipients) mail.Bcc.Add(MailboxAddress.Parse(bccRecipient));
+            foreach (var bccRecipient in bccRecipients)
+                mail.Bcc.Add(MailboxAddress.Parse(bccRecipient));
+
+            if (headers != null && headers.Length > 0)
+                foreach (var header in headers)
```

Comment:
In all honesty, this nested if-foreach without any brackets looks really dangerous if someone would ever to add more code and the readability is somewhat flaky to my eye. I beg you to consider adding at least brackets for the `if` statement.